randovania/games/am2r/logic_database/Industrial Complex.txt

```diff
@@ -1109,18 +1109,21 @@ Extra - minimap_data: [{'x': 67, 'y': 29}, {'x': 67, 'y': 30}, {'x': 68, 'y': 28
   * 4-Tiles High Dock to Lower Factory Lava Lake Access/Horizontal Dock to Lower Factory Intersection
   > Horizontal Dock to Lower Factory Hub
       Any of the following:
-          Space Jump or Can Use Spider Ball
+          Space Jump
           All of the following:
               Hi-Jump Boots
               Power Grip or Walljump (Beginner)
           # IBJ
           Infinite Bomb Jumping (Intermediate) and Can Use Bombs
           # CBJ
           Charged Bomb Jump (Intermediate) and Can Use Charged Bomb Jump
+          All of the following:
+              Can Use Spider Ball
+              Bombs or Spring Ball or Mid-Air Morph (Beginner)
```

Comment:
Is there really no way to do this via the accompanying rooms? :(
I assume the shirks hit you if you do this? 
![grafik](https://github.com/randovania/randovania/assets/38186597/0737bec5-21e9-4636-a51c-3cacdcfed98d)

And this seems a bit too much
![grafik](https://github.com/randovania/randovania/assets/38186597/66283afb-6231-42b0-a61b-763246bfb368)

Reply:
I tried both to see if movement was possible and it is not, the only thing that I can think of would be use a PB as soon as you enter the room.